### PR TITLE
feat: added home and product page

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Live preview link: 
https://brand-factory-react.netlify.app/

### CODE REVIEW

File: src/pages/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import {Nav} from "./../../components/Nav/Nav";
+import Hero_Img from './../../assets/hero2.jpg';
+import {Card} from './../../components/Card/Card';
```
Comment (dusariudaykumar) [inline src/pages/HomePage/HomePage.jsx:4]: One suggestion You can use an index.js file and add all your exports here and you can simply import all your component from a single file.

**Review by dusariudaykumar — APPROVED** [review]
The UI looks awesome.
The code is clean and great folder structure.

File: src/pages/HomePage/HomePage.jsx (hunk 1 above)
Comment (bishnoimukesh) [reply src/pages/HomePage/HomePage.jsx:4]: Thanks for the suggestion.
I will implement this.

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

**Pull request merged**